base_vat_optional_vies/models/res_partner.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3: Antiun Ingenieria S.L. - Antonio Espinosa
+# See README.rst file on addon root folder for more details
+
+import logging
+import re
+_logger = logging.getLogger(__name__)
+
+try:
+    import vatnumber
+except ImportError:
+    _logger.warning(
+        "VAT validation partially unavailable because the `vatnumber` Python "
+        "library cannot be found. Install it to support more countries, "
+        "for example with `easy_install vatnumber` or "
+        "`pip install vatnumber`.")
+    vatnumber = None
+
+from openerp import models, fields, api, _
+from openerp.exceptions import ValidationError
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    vies_passed = fields.Boolean(
+        string="VIES validation passed", readonly=True)
+
+    def __init__(self, pool, cr):
+        super(ResPartner, self).__init__(pool, cr)
+        self._constraints = []
```

Comment:
I remove them in order to overwrite ```check_vat``` constraint.
https://github.com/odoo/odoo/blob/8.0/addons/base_vat/base_vat.py#L160

I don't know any other way to do it. Do you know a more elegant way?